workshop/exercise-2/README.md

```diff
@@ -14,21 +14,52 @@ The application for this lab is a simple node.js with express app which returns
 	cd fib-knative
 	```
 
+## Set up a Private Container Registry and Obtain Credentials
+In this lab, we'll use the IBM Container Registry to host our docker images since you already have access to this through your IBM Cloud Account. IBM Container Registry enables you to store and distribute container images in a fully managed private registry.
```

Comment:
s/docker images/container images/

minor thing but I'd like to get into the habit of not using "docker" any more since docker may not be in the picture any more

Reply:
Let's also convert one of those "IBM Container Registry" phrase to a link (e.g. https://console.bluemix.net/docs/services/Registry/registry_overview.html#registry_overview ) so people can find one more.

Reply:
let's modify all of the exercises to use "container images" instead of "docker images"